Makefile.in

```diff
@@ -121,5 +118,13 @@ install:
 	$(INSTALL) -m 644 $(SRC_LDFILE) $(DESTDIR)$(target_libdir)/$(LDFILE)
 
 sinclude .deps
-.deps:
+.deps: .stamp-local-include
 	$(CC) -MM $(CPPFLAGS) $(srcdir)/*.c > .deps
+
+.stamp-local-include:
+	ln -sf $(BFD_INCLUDE_DIR)/bfd.h bfd.h
```

Comment:
i think we should create a subdir like bfd-headers/ that we can then `rm -rf` during the clean phase